src/axom/sidre/core/DataStore.hpp

```diff
@@ -540,6 +549,13 @@ class DataStore
 
   /// Flag indicating whether SLIC logging environment was initialized in ctor.
   bool m_need_to_finalize_slic;
+
+  /// Flag indicating a Conduit error occurred (consult message for details)
+  /// Instead of a boolean flag, we could store an int, an enum, even a bitfield.
+  mutable bool m_conduit_error_occurred;
+
+  /// Details of the most recent Conduit error
+  mutable std::string m_conduit_errors;
```

Comment:
The errors and the error flag need to be `mutable` so we can store values here even from within a `const` method like `Group::save()`.